codespell_lib/data/dictionary.txt

```diff
@@ -22818,6 +22818,8 @@ Eurpean->European
 Eurpoean->European
 evailable->available
 evalation->evaluation
+evaliate->evaluate
```

Comment:
May I suggest all verbal forms (`evaliated`, `evaliates`, `evaliating`)?

Reply:
Can you find any typos of `validate`/`revalidate`?

Reply:
Sure!